packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -735,6 +735,21 @@ export default class Meeting extends StatelessWebexPlugin {
    */
   setUpLocusMediaSharesListener() {
     this.locusInfo.on(EVENTS.LOCUS_INFO_UPDATE_MEDIA_SHARES, (payload) => {
+      // selfId !== activeConnectSharinId && isSharing == true
+      /**
+       * if (ndedCOntentSharingId === 'Released' && isSharing) stopShare()
+       */
+
+      const {contentId, disposition} = payload.current;
+
+      if (
+        this.isSharing &&
+        this.selfId !== contentId &&
+        disposition !== FLOOR_ACTION.RELEASED
+      ) {
+        this.stopShare();
```

Comment:
this.stopShare just makes the api call for the floor grant service but you also need to do updateShare(sendShare:false, receiveShare:true).

I guess with you just do updateShare it should handle the `this.stopShare` logic as well